app/src/main/java/com/vero/tasky/agenda/data/repository/AgendaRepositoryImpl.kt

```diff
@@ -69,6 +69,10 @@ class AgendaRepositoryImpl(
         saveAgendaItems(result)
     }
 
+    override suspend fun logOut() = safeApiCall {
+        api.logOut()
```

Comment:
On logout, you could also delete all the tables from the local db

Mostly because if the user logs in with another account, you wouldn't want them to access the other user's calendar